R/pkg/R/context.R

```diff
@@ -451,3 +435,31 @@ setCheckpointDir <- function(directory) {
   sc <- getSparkContext()
   invisible(callJMethod(sc, "setCheckpointDir", suppressWarnings(normalizePath(directory))))
 }
+
+#' Total number of CPU cores of all executors registered in the cluster at the moment.
+#'
+#' @rdname spark.numCores
+#' @return current number of cores in the cluster.
+#' @examples
+#'\dontrun{
+#' spark.numCores()
+#'}
+#' @note numCores since 2.4.0
+spark.numCores <- function() {
```

Comment:
you don't have to call it spark.*
i'm not sure what names is better, spark.numCores or numCores.